integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/ClusterUtils.java

```diff
@@ -67,28 +67,28 @@ public static ClusterResource createClusterResource(String clusterResName,
    */
   public static void createClusterAndVerify(ClusterResource cluster) {
     LoggingFacade logger = getLogger();
-    String clusterName = cluster.getSpec().getClusterName();
+    String clusterResourceName = cluster.getClusterResourceName();
     String namespace = cluster.getNamespace();
     // create the cluster CR
     assertNotNull(cluster, "cluster is null");
```

Comment:
This is not going to fail, if cluster is null line # 70 will throw NPE
You can move line 70 and 71 below these null checks